tests/migrate/test_plural.py

```diff
@@ -204,7 +254,8 @@ def test_too_few_variants(self):
                     { $num ->
                         [one] Usunąć plik?
                         [few] Usunąć { $num } pliki?
-                       *[many] Usunąć { $num } plików?
+                        [many] Usunąć { $num } plików?
+                       *[other] Usunąć { $num } plików?
```

Comment:
`compare-locales` lists [`one, few, many`](https://hg.mozilla.org/l10n/compare-locales/file/tip/compare_locales/plurals.py#l29) for Polish. I was caught off-guard seeing `other` in this test. Can you change the test setup to `plural_categories = ('one', 'few', 'many')` and the Polish string to `deleteAll=Usunąć plik?;Usunąć #1 pliki?` to simulate too few variants? The comment in `test_too_few_variants` needs adjusting, too.